avm/res/document-db/database-account/version.json

```
@@ -1,6 +1,6 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.6",
+    "version": "1.0",
```

Hi @seesharprun please consider we are not yet releasing modules at version 1.0, so breaking changes are still to use minor updates together with backward compatible new features. Ref https://azure.github.io/Azure-Verified-Modules/specs/shared/#module-versioning-before-first-major-version-release-100

```suggestion
    "version": "0.7",
```